lib/active_model_serializers.rb

```diff
@@ -44,6 +44,7 @@ def silence_warnings
 require 'active_model/serializer'
 require 'active_model/serializable_resource'
 require 'active_model/serializer/version'
+require 'active_model/serializer/assertions'
```

Comment:
Thanks for the feedback.

I think if we try to autoload `ActiveModel::Serializer::Assertions`, this will be loaded when we mix this on the `ActionController::TestCase`, in this same file, so this wil be loaded too soon.
Do you have some idea how we can lazy load this?